superset/migrations/versions/02f260780a2b_add_tables_for_sql_lab_state.py

```diff
@@ -0,0 +1,87 @@
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Add tables for SQL Lab state
+
+Revision ID: 02f260780a2b
+Revises: def97f26fdfb
+Create Date: 2019-08-12 14:50:49.204635
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "02f260780a2b"
+down_revision = "def97f26fdfb"
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "tab_state",
+        sa.Column("created_on", sa.DateTime(), nullable=True),
+        sa.Column("changed_on", sa.DateTime(), nullable=True),
+        sa.Column("extra_json", sa.Text(), nullable=True),
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("user_id", sa.Integer(), nullable=True),
+        sa.Column("label", sa.String(length=256), nullable=True),
+        sa.Column("active", sa.Boolean(), nullable=True),
+        sa.Column("database_id", sa.Integer(), nullable=True),
+        sa.Column("schema", sa.String(length=256), nullable=True),
+        sa.Column("query_id", sa.Integer(), nullable=True),
+        sa.Column("query_limit", sa.Integer(), nullable=True),
+        sa.Column("autorun", sa.Boolean(), nullable=True),
+        sa.Column("template_params", sa.Text(), nullable=True),
+        sa.Column("created_by_fk", sa.Integer(), nullable=True),
+        sa.Column("changed_by_fk", sa.Integer(), nullable=True),
+        sa.ForeignKeyConstraint(["changed_by_fk"], ["ab_user.id"]),
+        sa.ForeignKeyConstraint(["created_by_fk"], ["ab_user.id"]),
+        sa.ForeignKeyConstraint(["database_id"], ["dbs.id"]),
+        sa.ForeignKeyConstraint(["query_id"], ["query.id"]),
+        sa.ForeignKeyConstraint(["user_id"], ["ab_user.id"]),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_table(
+        "table_schema",
+        sa.Column("created_on", sa.DateTime(), nullable=True),
+        sa.Column("changed_on", sa.DateTime(), nullable=True),
+        sa.Column("extra_json", sa.Text(), nullable=True),
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("tab_state_id", sa.Integer(), nullable=True),
+        sa.Column("database_id", sa.Integer(), nullable=False),
+        sa.Column("schema", sa.String(length=256), nullable=True),
+        sa.Column("table", sa.String(length=256), nullable=True),
+        sa.Column("description", sa.Text(), nullable=True),
```

Comment:
@etr2460 I renamed `results` to `description`, since `metadata` is reserved in SQLAlchemy.